package.json

```diff
@@ -10,10 +10,7 @@
   "module": "dist/esm/index.js",
```

Comment:
Did you forget to update the entrypoints and exports map?

Reply:
yes, i forgot!. I'll fix it